fgd/point/info/info_null.fgd

```diff
@@ -9,3 +9,10 @@
 	[
 	@resources []
 	]
+
+// works identically to mbase's func_null when as a brush, so
+@SolidClass base(BaseEntity)
+= info_null: "A brush entity that's immediately removed upon spawning. Useful for invisible brush lights."
```

Comment:
I’m not sure if having a `SolidClass` and `PointClass` with the same name will actually work, both my code and I think Valve’s will have one definition overwrite the other.

Reply:
hmm, good point. I was thinking of trying a hacky `BaseClass` to fix it, but the only use I can find for that to transfer properties (usually models) to entities...

Reply:
I can do it in the postcompiler though, enable func_null if srctools is present, then swap the classname in the bsp.

Reply:
That could work :D